src/Microsoft.AspNetCore.ResponseCaching/ResponseCache/IResponseCache.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.ResponseCaching
+{
+    public interface IResponseCache
+    {
+        object Get(string key);
+        void Set(string key, object entry);
```

Comment:
Yup, I added a TODO comment in `DistributedResponseCache` for it to track. Maybe I should move it here since it also affects IMemoryCache as well.